nekoyume/Assets/_Scripts/State/RxProps.Arena.cs

```diff
@@ -400,7 +400,7 @@ var participants
                     avatar,
                     (win, lose)
                 );
-            }).ToArray();
+            }).OrderByDescending(e => e.Score).ThenByDescending(e => e.AvatarAddr == currentAvatarAddr).ToArray();
```

Comment:
Is it necessary to use the `OrderByDescending(e => e.Score)` method?
The `AddRand()` method performs a sorting according to the score and the `avatarAddrAndScoresWithRank` variable is populated with its return value.
I just curious.

```cs
L325:    var avatarAddrAndScoresWithRank =
L326:                     AddRank(avatarAddrAndScores);
```

Reply:
I fixed it, Could you check it again?